### ISSUE TITLE
Support Request for Unicode Characters

### ISSUE THREAD

**Issue by malulleybovo** [protectwise/troika#140]
Thank you for open-sourcing such an amazing web graphics tool! I've decided to include it in a new project of mine out of all the available options due to performance and ability to anchor text to 3D Objects in a Scene.

However, although not major, I'd like to have Unicode support in this tool. For example, I tried to render japanese text (stuff other than english) and ended with that generic square glyph that you always get for unsupported characters. Is this viable? Would you consider adding such support?

This would certainly increase the value of the tool (as long as it doesn't take a big toll on performance). Should anyone else agree with this, you're welcome to join the thread.

Thank you once again for the great project.

**Comment by lojjic** [comment]
Hi, thanks for posting. Unicode characters should be fully supported, but you have to give it a font that includes those characters. (The default font only includes a basic character set.)

See issue #13 for an enhancement that would allow dynamic loading of separate font files for different unicode ranges.

**Comment by malulleybovo** [comment]
@lojjic you were right! I did some more digging in the documentation and found what I wanted at https://www.npmjs.com/package/troika-three-text in the "preloading" section. For anyone else that may stumble upon this issue, look into `function preloadFont({ font: 'path/to/some/supported/font/file.woff' })` in the documentation; plus `someTextInstance.font = 'path/to/some/supported/font/file.woff'`.
I imported a font with a broader glyph set and it worked wonders out of the box with the above, thumbs up for that.
